crud/src/main/java/com/redhat/lightblue/crud/mongo/MongoCRUDController.java

```diff
@@ -349,11 +347,10 @@ public CRUDFindResponse find(CRUDOperationContext ctx,
                 ctx.addError(Error.get(MongoCrudConstants.ERR_NO_ACCESS, "find:" + ctx.getEntityName()));
             }
         } catch (Error e) {
-            // rethrow lightblue error
-            throw e;
+            ctx.addError(e);
```

Comment:
We have logging in some of the Error.get methods. Lets add logging to all
Error.get methods, so these would be logged everywhere.

On Thu, Jul 9, 2015 at 12:59 PM, Marek <notifications@github.com> wrote:

> In
> crud/src/main/java/com/redhat/lightblue/crud/mongo/MongoCRUDController.java
> <https://github.com/lightblue-platform/lightblue-mongo/pull/130#discussion_r34290665>
> :
>
> > @@ -349,11 +347,10 @@ public CRUDFindResponse find(CRUDOperationContext ctx,
> >                  ctx.addError(Error.get(MongoCrudConstants.ERR_NO_ACCESS, "find:" + ctx.getEntityName()));
> >              }
> >          } catch (Error e) {
> > -            // rethrow lightblue error
> > -            throw e;
> > +            ctx.addError(e);
>
> Why isn't the error logged (LOGGER.error) here? It's very helpful to have
> all errors logged lightblue side, clients may not log them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-mongo/pull/130/files#r34290665>
> .
>